schemas/audiences/segment.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/audiences/segment",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Assignment",
+  "type": "object",
+  "description":
+    "",
+  "definitions": {
+    "segment": {
+      "properties": {
+        "@id": {
```

Comment:
As this is not the full segment or audience, but only the assignment, it might be better to make this a `@type` (or introduce a `@type` in addition to the `@id`)